fastlane/Fastfile

```diff
@@ -37,6 +37,59 @@ platform :android do
     bump(options)
     attach_changelog_to_master
   end
+
+  desc "Make github release"
+  lane :github_release do |options|
+    release_version = options[:version]
+    fail ArgumentError, "missing version" unless release_version
+
+    begin
+      changelog = File.read("../CHANGELOG.latest.md")
+    rescue
+      fail "please add a CHANGELOG.latest.md file before calling this lane"
+    end
+
+    set_github_release(
+      repository_name: "revenuecat/purchases-android",
+      api_token: ENV["GITHUB_TOKEN"],
+      name: release_version,
+      tag_name: "#{release_version}",
+      description: changelog,
+      commitish: "master",
+      upload_assets: [],
+      is_draft: true
+  )
+  end
+
+  desc "Deploy a Snapshot release"
+  lane :deploy_snapshot do | options |
+    gradle_file_path = "purchases/build.gradle"
+    version = android_get_version_name(gradle_file: gradle_file_path)
+    is_snapshot_version = version.end_with?("-SNAPSHOT")
+    if is_snapshot_version
+      deploy
+    else
+      puts "purchases/build.gradle doesn't have a SNAPSHOT version"
```

Comment:
this is perfect, so this is just an FYI comment - ruby also has an alternate syntax that's similar to `guard`s in swift:
```ruby
puts "purchases/build.gradle doesn't have a SNAPSHOT version" unless is_snapshot_version
deploy
```
or 

```ruby
unless is_snapshot_version then puts "purchases/build.gradle doesn't have a SNAPSHOT version" end
deploy
```